doc/nrf/ug_avsystem.rst

```diff
@@ -0,0 +1,199 @@
+#. Optional integration of nRF Cloud with AVSystem for utilizing location services.
+
+.. rst-class:: numbered-step
+
+Adding your device to Coiote Device Management server
+=====================================================
+
+To add your device to AVSystem's Coiote Device Management Server, complete the following steps:
+
+.. include:: includes/lwm2m_common_server_setup.txt
+   :start-after: coiote_start
+   :end-before: coiote_end
+
+.. rst-class:: numbered-step
+
+Configuring your application to use Coiote Device Management platform
+=====================================================================
+
+For the device-server integration, you must select an appropriate sample or application that supports LwM2M-based interaction with AVSystem.
+It is then built and flashed on to the device with appropriate configurations.
+See :ref:`avsystem_supported_applciations` for a list of supported samples and applications.
+To integrate your device with AVSystem's device management platform, complete the following steps.
+
+a. |connect_kit|
+#. |connect_terminal|
+#. `Configure, build, and flash the supported sample or application using nRF Connect for VS Code <How to build an application_>`_.
+
+   .. note::
+
+      While building, make sure to select the appropriate overlay file for AVSystem integration.
+      This can vary for each sample or application.
+      For example, for :ref:`lwm2m_client` sample, you need to select the overlay file :file:`overlay-avsystem.conf` as the Kconfig fragment while configuring the sample.
+      Refer to the **Configuration files** section in the respective sample or application documentation for more information.
+
+After flashing the sample, reboot your device for reconnection.
+From the terminal, you can see that the registration is complete.
+You can also check the status on the Coiote Device Management UI.
+On the `Coiote Device Management UI <Coiote Device Management server_>`_, after adding your device, you can see a ``Device connected successfully`` message if the device was added successfully.
+Click :guilabel:`Next`, :guilabel:`Finish` and :guilabel:`Go to device`.
+You can see that the device is shown as registered under :guilabel:`Registration status` in **Device Center**.
+
+nRF Cloud integration with AVSystem
+***********************************
+
+You can optionally integrate nRF Cloud with AVSystem and make use of nRF Cloud Location services for device positioning.
+See `nRF Cloud integration with Coiote Device Management`_ for more information on how the integration can be done.
+
+The integration supports the following location services:
+
+* Cell-based location assistance
+* A-GPS location assistance
+
+Cell-based location
+===================
+
+You can use a supported sample to determine the cell-based location of the device if you have nRF Cloud integration with AVSystem.
+Cell-based location offers accuracy up to 1000 meters.
```

Comment:
In the case of single- or multicell? I assume that at least with multicell the accuracy is better than a kilometer.

Reply:
Removed this as it is already explained in https://docs.nrfcloud.com/LocationServices/LocationServicesOverview/#types.